pkgs/applications/misc/bottles/vulkan_icd.patch

```diff
@@ -1,11 +1,13 @@
 diff --git a/bottles/backend/utils/vulkan.py b/bottles/backend/utils/vulkan.py
-index 6673493..07f70d1 100644
+index 6673493..9191004 100644
 --- a/bottles/backend/utils/vulkan.py
 +++ b/bottles/backend/utils/vulkan.py
-@@ -29,6 +29,8 @@ class VulkanUtils:
+@@ -28,7 +28,9 @@ class VulkanUtils:
+         "/usr/share/vulkan",
          "/etc/vulkan",
          "/usr/local/share/vulkan",
-         "/usr/local/etc/vulkan"
+-        "/usr/local/etc/vulkan"
++        "/usr/local/etc/vulkan",
 +        "/run/opengl-driver/share/vulkan/",
 +        "/run/opengl-driver-32/share/vulkan/",
      ]
```

Comment:
@mirkobrombin should we upstream this?

Reply:
I think it's convenient

Reply:
No, this paths are exclusive to NixOS.

Reply:
Sure, but upstreaming it would stop the need from creating a patch, no? I think it should be in upstream.

Reply:
upstream only really wants to support their flatpack distribution so I don't think we need to invest much time here.